app/src/time-of-day/time-off-day.controller.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    Controller,
+    Get,
+    Response,
+    HttpStatus,
+    Query,
+    Post,
+    Body,
+    Patch,
+    Param,
+    UseGuards,
+    Headers,
+    UnauthorizedException,
+    Delete,
+} from '@nestjs/common';
+
+import { AuthGuard } from '@nestjs/passport';
+import { AuthService } from '../auth/auth.service';
+import { TimeOffDayService } from './time-off-day.service';
+
+@Controller('time-off-day')
+export class TimeOffDayController {
+    constructor(private readonly authService: AuthService, private readonly timeOffDayService: TimeOffDayService) {}
+
+    @Post('add')
+    @UseGuards(AuthGuard())
+    async createTimeOffDay(@Headers() headers: any, @Response() res: any, @Body() body: any) {
+        const createdById = await this.authService.getVerifiedUserId(headers.authorization);
+        if (!createdById) {
+            throw new UnauthorizedException();
+        }
+
```

Comment:
Missed check for body params existence. Look at the Resource Planning Controller to find examples